python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Events.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+    LEDBasicState_Unknown=0,
+    LEDBasicState_Off=1,
+    LEDBasicState_On=2,
+    LEDBasicState_UnexpectedlyOn=7,
+    LEDBasicState_UnexpectedlyOff=8
+  </Enumeration>
+  <Enumeration>
+    LabjackControllerState_Unknown=0,
+    LabjackControllerState_StandbyOrOn=1,
+    LabjackControllerState_Cooldown=2,
+    LabjackControllerState_Error=3
+  </Enumeration>
+  <SALEvent>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_logevent_labjackConnected</EFDB_Topic>
```

Comment:
Consider merging this event with `labjackState` below. Basically, add `connected` to the event. However, you could also create a new `Disconnected` state or assume `Unknown` means disconnected.